src/actions/index.js

```diff
@@ -458,10 +501,108 @@ function updateOutput(
         });
     }
 
+    // Clientside hook
+    if (!isNil(clientside_function) && !isEmpty(clientside_function)) {
+        let returnValue;
+        try {
+            returnValue = window[clientside_function.namespace][
+                clientside_function.function_name
+            ](
+                ...(has('inputs', payload)
+                    ? pluck('value', payload.inputs)
+                    : []),
```

Comment:
For what it's worth, [tested](https://jsperf.com/ramdavanilla) that doing this type of operation with Ramda instead of using a plain js tertianary `payload.inputs ? payload.input.value : []) is about 3 orders of magnitude slower (~1750x). Haven't tested with all usages here but pretty sure this comment will apply to other similar usages as well.

Reply:
Those tests aren't implementing `pluck` correctly, they are actually implementing `prop` https://ramdajs.com/docs/#prop https://ramdajs.com/docs/#pluck

Reply:
I'm not sure I understand. The above uses `pluck` the linked test uses `pluck` not `prop`.
![image](https://user-images.githubusercontent.com/8092993/55401115-c6d87d80-551d-11e9-908e-0cbbeca88722.png)

Reply:
Your vanilla code implements `R.prop` and compares it with `R.pluck` called on the wrong data shape (`R.pluck` does something... interesting... when called on an object rather than an array, and maybe that's part of its relative slowness)

It's certainly concise ⛳️ to do this with ramda - I'm curious though, is there actually a case for no `inputs` anymore or is that a relic of `events` days?

Reply:
I see. You're right, I'm comparing apples and oranges. The Ramda performance is more than fine when testing the right thing.

Reply:
>  I'm curious though, is there actually a case for no inputs anymore or is that a relic of events days?

there is not. good catch!